### PR TITLE
February 23 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 9172282 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-402.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+|--------------------------------------------|--------------------------|
+| Axway API Management 7.6.2 SPx and 7.7 SPx | DA=v1.1.71 / TA=v1.1.73  |
+| AWS Gateway using SDK 2.0                  | DA=v1.1.59 / TA-v1.1.61  |
+| Azure latest release                       | DA=v1.1.65 / TA=v1.1.70  |
+| Istio 1.9.5                                | DA=v1.0.31 / TA=v2.0.75  |
+| Apigee Edge                                | 0.1.31                   |
+| Mulesoft Anypoint platform v3              | 1.1.42                   |
+| Software AG webMethods                     | 0.0.19                   |
+| Kong Gateway                               | 1.0.2                    |
+| GitLab                                     | 1.0.1                    |
```
Comment (vivekschauhan) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:27]: 1.0.2

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+| Apigee X Embedded Service                  |
+| GitHub Embedded Service                    |
+
+### Axway Agent SDK
+
+The following enhancements are available in this release.
+
+#### Enhancements for Agent SDK
+
+* **Seccrity Methods specification** - The SDK has been updated to enable the Agent to configure security.  This is used to enable the Try-it capability for Kong Gateway.
```
Comment (vivekschauhan) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:42]: Seccrity -> Security

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+
+#### Bug fixes for Agent SDK
+
+* **Traceability Agent Usage Reporting** - The usage reporting file contains a granularity interval which indicates the reporting period.   This was fixed to report the correct granularity instead of alway reporting 15 minutes.
```
Comment (vivekschauhan) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:46]: alway -> always

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+The following bug fixes are available in this release.
+
+* **Memory Usage Increase** - A fix was made to the prevent the increased memory usage over time with Traceability Agent to v1.1.69.
+* **Traceability Agent Upgrade Issue** - A helm chart override was added to fix an issue when upgrading the Traceability Agent to v1.1.69.
+
+##### Known issues for Axway API Management Agent
+
+There are no known limitations in this release.
+
+#### Mulesoft Gateway Agent
```
Comment (vivekschauhan) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:73]: "Kong Agent" I guess

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+
+There are no new enhancements or bug fixes for any SaaS (embedded) Agents in this release.
+
+### Axway CLI
+
+The latest version of the [Axway CLI is v2.0.16 on NPM](https://www.npmjs.com/package/@axway/axway-cli/v/2.0.16). There are no new enhancements or bug fixes for Axway CLI in this release.
+
+### Axway Central CLI
+
+The latest version of the [Axway Central CLI is v2.25.0 on NPM](https://www.npmjs.com/package/@axway/axway-central-cli/v/2.25.0). There are no new enhancements or bug fixes for Axway Central CLI in this release.
```
Comment (vivekschauhan) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:117]: Should we change this "There are no new enhancements or bug fixes for Axway Central CLI in this release" as there are new feature update and bug fix

File: content/en/docs/amplify_relnotes/_index.md [hunk 6]
```diff
@@ -20,6 +20,55 @@ To view the *Administration Release Notes* [click here](https://docs.axway.com/b
...
+
+* **Axway API Management 7.6.2 SPx and 7.7 SPx** - DA=v1.1.70 / TA=v1.1.72
+* **AWS Gateway using SDK 2.0** - DA=v1.1.58 / TA-v1.1.60
+* **Azure latest release** - DA=v1.1.64 / TA=v1.1.69
+* **Istio 1.9.5** - DA=v1.0.30 / TA=v2.0.74
+* **Apigee Edge** - Agent version 0.1.30
+* **Mulesoft Anypoint platform v3** - Agent version 1.1.40
+* **Software AG webMethods** - 0.0.18
+* **Kong Gateway** - 1.0.1
+* **GitLab** - 1.0.1
```
Comment (vivekschauhan) [inline content/en/docs/amplify_relnotes/_index.md:35]: Version seems to be from previous release and Kafka is missing

File: content/en/docs/amplify_relnotes/20240223_amplify.md (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240223_amplify.md:27]: Ah Gitlab is 1.0.2 , Dale started that one late :)

File: content/en/docs/amplify_relnotes/20240223_amplify.md (hunk 3 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240223_amplify.md:46]: fixed

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+
+##### Known issues for Kafka Cluster Agent
+
+There are no known limitations in this release.
+
+#### Kong Gateway Agent
+
+The following enhancements are available in this release.  There are no known defects or limitations in this release.
+
+##### Enhancements for Kong Gateway Agent
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240223_amplify.md:99]: I removed the mulesoft enhancement because it was really for Kong (the security setting)

**Review by vivekschauhan — APPROVED** [review]

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+| Kong Gateway                               | 1.0.2                    |
+| GitLab                                     | 1.0.2                    |
+| Kafka Cluster                              | DA=v1.0.0 / TA=v1.0.0    |
+
+| Supported SaaS (embedded) Agents                      |
+|--------------------------------------------|
+| AWS Embedded Service                       |
+| Apigee X Embedded Service                  |
+| GitHub Embedded Service                    |
+
```
Comment (cbordier) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:35]: Maybe we could indicate there that now agents are visible from any organization:
- from Help menus > Downloads > Repository and then search for agents.
OR 
- from the following url: https://repository.axway.com/catalog?q=agents

Comment (georgiana-caras) [reply content/en/docs/amplify_relnotes/20240223_amplify.md:35]: I like this suggestion.

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 9]
```diff
@@ -0,0 +1,206 @@
...
+There are no known limitations in this release.
+
+### Product Foundry
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+* **Customize the product versioning** - When releasing a product, it is now possible to add your own custom product version, a short description as well as to automatically archive previous version. [Refer to activate a product release - step 7.](/docs/manage_product_foundry/foundry_product_management#actiovate-a-product-release). Then you can decice with the [Marketplace settings - Custom Version Display](/docs/manage_marketplace/customize_marketplace/marketplace_settings#configure-settings) what the consumer should see: either the custom version or the generated one.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:169]: New settings have been added to the Release Product version screen to allow providers to add a custom product version. The display of the custom product version in a Marketplace can be controlled through a setting on the [Marketplace Settings screen](/docs/manage_marketplace/customize_marketplace/marketplace_settings#configure-settings). In addition to providing a custom product version, the user can choose to archive older version and add a short description to indicate what is included in the new version. [Refer to activate a product release - step 7](/docs/manage_product_foundry/foundry_product_management#actiovate-a-product-release) for more details.

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 10]
```diff
@@ -0,0 +1,206 @@
...
+* **Product Ratings** - Provider can turn on the Rating feature for a specific Marketplace. Refer to [Marketplace settings - Ratings](/docs/manage_marketaplace/customize_marketplace/marketplace_ratings_reviews). On the product details screen, provider will see all the ratings available for his product. Refer to [Product rating management](/docs/manage_product_foundry/foundry_product_ratings_reviews).
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:170]: Remove this section.

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 11]
```diff
@@ -0,0 +1,206 @@
...
+
+There are no new features or bug fixes in this release.
+
+## Consumer experience
+
+### Marketplace
+
+#### Feature updates for Marketplace
+
+* **Custom product version** - Based on provider preferences, the generated product version could be replaced by a custom version. Provider can also decide to limit the product history to only the latest version or the entire version list.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:190]: Remove the custom product version. It is enough to mention it under Product Foundry

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 12]
```diff
@@ -0,0 +1,206 @@
...
+* **Product rating** - When enabled by the provider, consumer can rate the product they have access to. Refer to [Rate a product](/docs/manage_marketplace/consumer_experience/ratings_reviews)
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:191]: **Product ratings** - Consumers can now rate their statisfaction with a product in the Marketplace. Marketplace Managers must turn on the Product Ratings feature in order for Consumer to be able to rate a product. Refer to [Marketplace settings - Ratings](/docs/manage_marketaplace/customize_marketplace/marketplace_ratings_reviews). Product owners can see the submitted ratings from the product details screen, in the Product Foundry.

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 13]
```diff
@@ -0,0 +1,206 @@
...
+
+#### Bug fixes for Marketplace
+
+* **Product release not present in Marketplace even if product is published and release is active** - A race condition when releasing a product version has been corrected to be able to see all the product releases on the Marketplace.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:195]: **Active product version not visible in a Marketplace for published products - An issue has been fixed to display all active product versions in a Marketplace.

File: content/en/docs/amplify_relnotes/20240223_amplify.md [hunk 14]
```diff
@@ -0,0 +1,206 @@
...
+* **Product details not visible in small screen resolution** - The product details screen is now responsive to the screen size.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240223_amplify.md:196]: **Product details not visible on small screens** - Users can now see the product details when they browse the product from screens with small resolution.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by cbordier — APPROVED** [review]
I approve.

**Pull request merged**